sse.go

```diff
@@ -139,6 +150,14 @@ func (es *EventSource) SetHeader(header, value string) *EventSource {
 	return es
 }
 
+// SetBody method sets body value to the [EventSource] instance
+//
+//	es.SetBody([]byte(`{"test":"put_data"}`),)
```

Comment:
@vegetablechicken233 This good example requires a correction based on method input type. Also, it has a syntax issue: `,` is present near the closing curve bracket.
```go
es.SetBody(strings.NewReader(`{"test":"put_data"}`))
```

Reply:
@jeevatkm  already fix it